src/parser/util.ts

```diff
@@ -114,6 +114,28 @@ export async function deflateGitObject(repo: string, hash: string) {
   return result as string
 }
 
+export async function getDefaultQuotePathValue(repoDir: string) {
+  const result = await runProcess(repoDir, "git", ["config", "core.quotepath"])
+  return result as string
+}
+
+export async function disableQuotePath(repoDir: string) {
+  await runProcess(repoDir, "git", ["config", "core.quotepath", "off"])
+  log.debug("Disabled quotePath")
+}
+
+export async function resetQuotePath(repoDir: string, value: string) {
+  log.debug("value: " +  value)
```

Comment:
Could this log be a bit more clear on what is logged?

Reply:
Oh, that should have been deleted, thanks for catching :D